subscription-service/src/main/kotlin/com/egm/stellio/subscription/model/Subscription.kt

```diff
@@ -24,4 +48,14 @@ data class Subscription(
             JsonLdUtils.expandJsonLdKey(it, context)!!
         }
     }
+
+    fun toJson(includeSysAttrs: Boolean): String {
```

Comment:
maybe a default set to `false` for `includeSysAttrs`?

Reply:
OK